java-engine/settings.gradle

```diff
@@ -6,6 +6,12 @@
  * Detailed information about configuring a multi-project build in Gradle can be found
  * in the user manual at https://docs.gradle.org/8.1/userguide/multi_project_builds.html
  */
+pluginManagement {
```

Comment:
No idea why this needed to happen but Gradle flat out refused to resolve jmh without it for some reason